backend/app/controllers/products.js

```diff
@@ -481,7 +481,8 @@ module.exports = {
             const fields = [ // List of CSV columns
                 'subject', 'user', 'surveyName', 'stage', 'question', 'questionType', 'response', 'choiceText',
                 'publicationLink', 'publicationTitle', 'publicationAuthor', 'publicationDate', 'commenter',
-                'commentReason', 'comment', 'date'
+                'commentReason', 'comment', 'historicCommenter1', 'historicReason1', 'historicComment1',
+                'historicCommenter2', 'historicReason2', 'historicComment2', 'date'
```

Comment:
Why are we capping this at 2 previous comments?

Reply:
This was from the suggestion @jmfadeley made on Slack earlier today I believe. 
"That includes the comment history and returns comments. It only goes back two histories ago. Technically I think the max length for commentHistory is four, discluding the current comment."